app/src/main/java/com/infomaniak/drive/ui/addFiles/NewFolderFragment.kt

```diff
@@ -85,16 +85,18 @@ class NewFolderFragment : Fragment() {
         }
     }
 
-    private fun initDropBoxFolder(drive: Drive?) {
+    private fun initDropBoxFolder(drive: Drive?) = with(binding) {
         val canCreateDropbox = drive?.canCreateDropbox == true
-        binding.myKSuitePlusChip.isVisible = !canCreateDropbox
-        binding.dropBox.apply {
+        myKSuitePlusChip.isVisible = !canCreateDropbox && drive?.isKSuitePersoFree == true
+        kSuiteProChip.isVisible = !canCreateDropbox && drive?.isKSuiteProFree == true
+        dropBox.apply {
             isVisible = drive?.sharedWithMe != true
             setOnClickListener {
+                val matomoName = "dropboxQuotaExceeded"
```

Comment:
Useless val

Reply:
Also Just thought about it, but it should not have any matomo "magic value" string, that's not in MatomoDrive.Category (or KSuiteMatomo)